### PR TITLE
Add Rancher example multi-user doc

### PR DESCRIPTION
**Pull request by raulcabello**
multi-user docs just cover fleet standalone. This PR introduces an example of how to achieve multi-user in Rancher by using `GlobalRoles` and `namespacedRules`.

### CODE REVIEW

File: docs/multi-user.md [hunk 1]
```diff
@@ -28,7 +28,7 @@ deploy resources without conflicts.
 
 :::
 
-## Example User
+## Example Fleet standalone
```
Comment (manno) [inline docs/multi-user.md:31]:
```suggestion
## Example Fleet Standalone 
```

File: docs/multi-user.md [hunk 2]
```diff
@@ -45,6 +45,79 @@ If we want to give access to multiple namespaces, we can use a single cluster ro
 
 This makes sure, tenants can't interfere with GitRepo resources from other tenants, since they don't have access to their namespaces.
 
+## Example Fleet in Rancher
+ 
+When a new fleet workspace is created, a corresponding namespace with an identical name is automatically generated within the Rancher local cluster.
+For a user to see and deploy fleet resources in a specific workspace needs at least the following permissions:
```
Comment (weyfonk) [inline docs/multi-user.md:51]: Are we talking about permissions that the _user_ must have? If so, I find this a bit clearer:
```suggestion
For a user to see and deploy fleet resources in a specific workspace, they need at least the following permissions:
```

File: docs/multi-user.md [hunk 3]
```diff
@@ -45,6 +45,79 @@ If we want to give access to multiple namespaces, we can use a single cluster ro
...
+- list/get the `fleetworkspace` cluster-wide resource in the local cluster
+- Permissions to create fleet resources (such as `bundles`, `gitrepos`, ...) in the backing namespace for the workspace in the local cluster. 
+
+Let's grants permissions to deploy fleet resources in the `project1` and `project2` fleet workspaces:
```
Comment (weyfonk) [inline docs/multi-user.md:55]:
```suggestion
Let's grant permissions to deploy fleet resources in the `project1` and `project2` fleet workspaces:
```

File: docs/multi-user.md [hunk 4]
```diff
@@ -45,6 +45,79 @@ If we want to give access to multiple namespaces, we can use a single cluster ro
...
+
+- To create the `project1` and `project2` fleet workspaces, you can either do it in the  [Rancher UI](https://ranchermanager.docs.rancher.com/integrations-in-rancher/fleet/overview#accessing-fleet-in-the-rancher-ui) or use the following YAML resources:
```
Comment (weyfonk) [inline docs/multi-user.md:57]: nit (removing extra space):
```suggestion
- To create the `project1` and `project2` fleet workspaces, you can either do it in the [Rancher UI](https://ranchermanager.docs.rancher.com/integrations-in-rancher/fleet/overview#accessing-fleet-in-the-rancher-ui) or use the following YAML resources:
```

**Review by weyfonk — COMMENTED** [review]
Nice one, thanks @raulcabello :)
Left a couple of nitpicks.

**Review by weyfonk — APPROVED** [review]

**Pull request merged**